src/v/iceberg/tests/compatibility_test.cc

```diff
@@ -1028,6 +1028,7 @@ class StructCompatibilityTestBase
         if (!raw.has_value()) {
             return partition_spec{};
         }
+        fmt::print(">>> RAW: {}\n", raw.value());
```

Comment:
nit: use a logger? Or is this left in from debugging?

Reply:
yeah, left from debugging